docs/protocol.md

```diff
@@ -257,92 +261,109 @@ def KGen():
     pk = (pk1, pk2)
     return (sk, pk)
 
-
-
 def AuthEnc(
-        skS1, skS2,  # (skS1, skS2)
-        pkR1, pkR2,  # (pkR1, pkR2)
-        m, ad, _info=None):
+        sk=(skS1, skS2),  # NB. invalid Python syntax for parity with the mathematical signature
+        pk=(pkR1, pkR2),
+        m, ad, info):
     (c2, K2) = KEM_PQ.Encap(pkR=pkR2)
-    (c1, cp) = pskAEnc(skS=skS1, pkR=pkR1, psk=K2, m=m, ad=ad, info=c2)  # cp = c'
+    (c1, cp) = pskAEnc(skS=skS1, pkR=pkR1, psk=K2, m=m, ad=ad, info=c2 + info)  # where cp = c' and "+" means concatenation
```

Comment:
[NB.](https://github.com/freedomofpress/securedrop-protocol/pull/126#issuecomment-3583554111) This isn't circular: `pskAEnc()`'s keyword argument `info` takes the concatenation of the local variable `c2` and the enclosing `AuthEnc()`'s positional argument `info`, which is where we pass $pk_R^{fetch}$:

https://github.com/freedomofpress/securedrop-protocol/blob/98511c28ee43d639d726eee88e4dfebce50a110c/docs/protocol.md?plain=1#L436